fades/main.py

```diff
@@ -58,8 +58,14 @@
 
 def _merge_deps(*deps):
     """Merge all the dependencies; latest dicts overwrite first ones."""
+    all_deps = []
     final = {}
     for dep in deps:
+        if isinstance(dep, dict):
```

Comment:
This kind of "dual type" consumption is tricky... it makes the code a little more complex here, but makes usage of this function a LOT more complex, as it's difficult to remember how to use it.

So, let's keep it simple. This function should receive a  list where each list is the dependency dict. IOW, the only change I'd do in this function is to receive "deps" (without the star), and let's change below how we call this...